### ISSUE TITLE
Missing Definition for some jsx-a11y rules

### ISSUE THREAD

**Issue by Fabs** [Shopify/javascript#51]
I have a meteor (1.3.2.4) project with react where I added eslint-plugin-shopify. 

All my files complain of missing jsx-a11y rules, on the first line (import React from 'react')
```
  1:1  error    Definition for rule 'jsx-a11y/img-uses-alt' was not found                    jsx-a11y/img-uses-alt
  1:1  error    Definition for rule 'jsx-a11y/label-uses-for' was not found                  jsx-a11y/label-uses-for
  1:1  error    Definition for rule 'jsx-a11y/mouse-events-map-to-key-events' was not found  jsx-a11y/mouse-events-map-to-key-events
  1:1  error    Definition for rule 'jsx-a11y/no-hash-href' was not found                    jsx-a11y/no-hash-href
  1:1  warning  Definition for rule 'jsx-a11y/redundant-alt' was not found                   jsx-a11y/redundant-alt
  1:1  error    Definition for rule 'jsx-a11y/valid-aria-role' was not found                 jsx-a11y/valid-aria-role

```

My package.json is:
```
{
  "name": "board",
  "version": "1.0.0",
  "description": "",
  "main": "index.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "lint": "eslint . --max-warnings 0"
  },
  "author": "",
  "license": "ISC",
  "dependencies": {
    "moment": "^2.13.0",
    "moment-timezone": "^0.5.3",
    "react": "^0.14.8",
    "react-dom": "^0.14.8",
    "react-redux": "^4.4.1",
    "react-router": "^2.0.1",
    "redux": "^3.5.2",
    "redux-thunk": "^2.0.1"
  },
  "eslintConfig": {
    "extends": [
      "plugin:shopify/react"
    ],
    "env": {
      "meteor": true,
      "node": true
    },
    "rules": {
      "no-warning-comments": "off",
      "no-console": "off",
    },
    "plugins": [
      "jsx-a11y"
    ]
  },
  "devDependencies": {
    "eslint": "^2.9.0",
    "eslint-plugin-babel": "^3.2.0",
    "eslint-plugin-import": "^1.6.1",
    "eslint-plugin-jsx-a11y": "^1.0.4",
    "eslint-plugin-shopify": "^11.1.2",
    "eslint-plugin-sort-class-members": "^1.0.1"
  }
}
```

So far, I have removed this lines on "rules" manually now, but I guess there is a problem with the way the shopify plugin uses jsx-a11y.

```
    "rules": {
      "no-warning-comments": "off",
      "no-console": "off",
      "jsx-a11y/img-uses-alt": "off",
      "jsx-a11y/label-uses-for": "off",
      "jsx-a11y/mouse-events-map-to-key-events": "off",
      "jsx-a11y/no-hash-href": "off",
      "jsx-a11y/redundant-alt": "off",
      "jsx-a11y/valid-aria-role": "off"
    },
```

**Comment by lemonmade** [comment]
Thanks for reporting the issue! I've just put up a PR to update the rule definitions for `jsx-a11y` to what they are in `1.0.4` (we were at `0.6.2`). You can see the progress here: #57.

**Comment by Fabs** [comment]
My pleasure. Thanks for the quick reply (and fix)! I will keep an eye on #57.
